react/src/components/nplusone.js

```diff
@@ -0,0 +1,53 @@
+import { Component } from "react";
+import Context from "../utils/context";
+import { Link, useFetcher } from "react-router-dom";
+import * as Sentry from "@sentry/react";
+import { connect } from "react-redux";
+import { determineBackendUrl } from "../utils/backendrouter";
+class NPlusOneItem extends Component {
+  async componentDidMount() {
+    console.log("Mounting NPlusOneItem");
+    const response = await fetch(this.props.backend + "/product/0/info?id=" + this.props.index);
+    console.log({ response });
+    const json = await response.json();
+    console.log({ json });
+    this.setState({
+      data: json,
```

Comment:
Does this `data` get used anywhere? I think not, and that's not necsarilly a problem. But I want to be aware what happens if a bunch of 500 responses get sent back (let's say the backend is down or just screwing up) - then what do we expect to happen with the NPlusOne component? Will it still render?

If it still renders, then that can be fine too, but dunno if it will stop the Perf Issue from occurring or any other erroneous outcomes we'll stumble upon later.

Reply:
Ya. We get back 500s since there isnt anything actually being sent back. but the page loads and the perf event triggers.